backend/exchanges/pocket.go

```diff
@@ -98,7 +98,7 @@ func GetPocketSupportedRegions(httpClient *http.Client) (map[string]PocketRegion
 	var regionsList []PocketRegion
 	endpoint := fmt.Sprintf("%s/availabilities", pocketAPILiveURL)
 
-	_, err := util.APIGet(httpClient, endpoint, "", 81920, &regionsList)
+	_, err := util.APIGet(httpClient, endpoint, "", 100000, &regionsList)
```

Comment:
This is too small of an increase I think. You can easily bump it 10x without any downsides and then we can be more confident there will not be a similar issue in the future.

Reply:
updated, yeah my increase was a bit more conservative.